app/src/components/Layers/TracksLayer.js

```diff
@@ -33,6 +33,7 @@ export default class TracksLayerGL {
    */
   _drawTrack(data, queriedSeries, hue, drawParams, offsets) {
     const color = hueToRgbHexString(hue);
+    const MIN_ZOOM_LEVEL = 5;
```

Comment:
Could you change the name to give it more context, like `TRACKS_DOTS_STYLE_ZOOM_THRESHOLD` for example, and move it to constants (see https://github.com/Vizzuality/GlobalFishingWatch/blob/develop/app/src/constants.js#L53)